libweb3jsonrpc/Test.cpp

```diff
@@ -33,6 +34,76 @@ using namespace jsonrpc;
 
 Test::Test(eth::Client& _eth): m_eth(_eth) {}
 
+Json::Value fillJsonWithState(eth::State const& _state, eth::AccountMaskMap const& _map)
+{
+    bool mapEmpty = (_map.size() == 0);
+    Json::Value oState;
+    for (auto const& a : _state.addresses())
+    {
+        if (_map.size() && _map.find(a.first) == _map.end())
+            continue;
+
+        Json::Value o;
+        if (mapEmpty || _map.at(a.first).hasBalance())
```

Comment:
Why is this empty case treated in a special way?

Reply:
this is actually a copy of the same method from tests. but that one works with json_spirit. 
I guess the account map here could be removed. because it is needed for expect sections with incomplete post states.